runtime.go

```diff
@@ -3229,3 +3236,7 @@ func (r *Runtime) getPrototypeFromCtor(newTarget, defCtor, defProto *Object) *Ob
 	}
 	return defProto
 }
+
+func (self *Runtime) DisableFunctionTickTracking() {
```

Comment:
I originally had a function to enable it but ending up removing because we are [checking the setting each function execution](https://github.com/10gen/baas/pull/12723/files#diff-b4c19cbed8278d92c3d7be77f467fefa2b7e8d4809862184c4506f21cef5dc21R412-R416).

However, that approach may change, I'll go ahead and re-add so we have that option moving forward.